query-node/mappings/src/content/nft.ts

```diff
@@ -797,31 +798,37 @@ export async function contentNft_AuctionBidCanceled({ event, store }: EventConte
 
   // specific event processing
 
-  // load video and auction
-  const { video, auction, nft } = await getCurrentAuctionFromVideo(
-    store,
-    videoId.toString(),
-    'Non-existing video got bid canceled',
-    'Non-existing NFT got bid canceled',
-    'Non-existing auction got bid canceled',
-    ['ownerMember', 'ownerCuratorGroup'],
-    ['topBid', 'bids', 'bids.bidder']
-  )
-
-  // retrieve relevant bid
-  const bid = (auction.bids || []).find((bid) => !bid.isCanceled && bid.bidder.id.toString() === memberId.toString())
+  const bid = await store.get(Bid, {
+    where: { bidder: { id: memberId }, nft: { id: videoId } },
+    relations: [
+      'nft',
+      'nft.video',
+      'nft.ownerMember',
+      'nft.ownerCuratorGroup',
+      'auction',
+      'auction.topBid',
+      'auction.bids',
+      'auction.bids.bidder',
+    ],
+  })
```

Comment:
Will this load the right bid though? Seems it only checks for bidder and video id, I think there can be multiple bids that can match this condition, probably should also filter by not cancelled bids?